VideoQuickStart/SceneDelegate.swift

```diff
@@ -10,7 +10,7 @@ import UIKit
 import TwilioVideo
 
 // Xcode 10.x will compile SceneDelegate even with the availability macro in place.
-#if XCODE_1100
+#if !XCODE_1000
```

Comment:
There is a similar problem with configuring the UserInterfaceTracker which relies on `#if XCODE_1100`. Can we just remove support for building on Xcode 10 at this time? Our SDK requires Xcode 11, so I think it is okay if the sample code does as well.

Reply:
Yes, that makes sense. For Twilio Video 3.x we already require developers to build their app with Xcode 11 so that the Bitcode is compatible. Removing the Xcode 10 macros altogether.